catalog.tests.bom

```diff
@@ -0,0 +1,164 @@
+        - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
+          name: Check PostgreSQL creating script has been executed
+          command: $brooklyn:formatString("psql -U postgres -p %s -qtA -c 'SELECT count(*) FROM users'", $brooklyn:entity("postgresqlnode").attributeWhenReady("postgresql.port"))
+          assertStatus:
+            equals: 0
+          assertOut:
+            matches: "2"
+          assertErr:
+            isEmpty: true
+
+      # Effector: restart Tests
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: "Effector: restart Tests"
+        brooklyn.children:
+        - type: org.apache.brooklyn.test.framework.TestEffector
+          name: Invoke restart Effector
+          effector: restart
+          params:
+            restartMachine: false
+          timeout: 5m
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check PostgreSQL is up after restart
+          sensor: service.isUp
+          timeout: 1m
+          assert:
+          - equals: true
+        - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
+          name: Check PostgreSQL process has restarted
+          command: ps -ef | grep [p]ostgres
+          assertStatus:
+            equals: 0
+          assertOut:
+            isEmpty: false
+          assertErr:
+            isEmpty: true
+
+      # Effector: stop + restart Tests
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: "Effector: stop + restart Tests"
+        brooklyn.children:
+        - type: org.apache.brooklyn.test.framework.TestEffector
+          name: Invoke stop Effector
+          effector: stop
+          params:
+            stopMachineMode: NEVER
+          timeout: 5m
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check PostgreSQL is not up after stop
+          sensor: service.isUp
+          timeout: 5m
+          assert:
+          - equals: false
+        - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
+          name: Check PostgreSQL process has stopped
+          command: ps -ef | grep [p]ostgres
+          assertStatus:
+            equals: 1
+          assertOut:
+            isEmpty: true
+          assertErr:
+            isEmpty: true
+        - type: org.apache.brooklyn.test.framework.TestEffector
+          name: Invoke restart Effector
+          effector: restart
+          params:
+            restartMachine: false
+          timeout: 5m
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check PostgreSQL is up after restart
+          sensor: service.isUp
+          timeout: 1m
+          assert:
+          - equals: true
+        - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
+          name: Check PostgreSQL process has restarted
+          command: ps -ef | grep [p]ostgres
+          assertStatus:
+            equals: 0
+          assertOut:
```

Comment:
I'd get rid of `assertOut` and `assertErr` here. If we trust the `SimpleShellCommandTest`, then the `assertStatus` of 0 should be enough, yes?

Same above (where we "Check PostgreSQL process has stopped")